mleap-databricks-runtime-fat-tests/README.md

```diff
@@ -0,0 +1 @@
+# MLeap Databricks ML Runtime Fat JAR test
```

Comment:
Why creating a new project? Can we put it under mleap-databricks-runtime or mleap-databricks-runtime-fat?